src/sentry/utils/silo/add_silo_decorators.py

```diff
@@ -27,9 +27,7 @@ def execute(
         predicate: Callable[[TargetClass], bool],
     ) -> None:
         filtered_targets = (
-            (c.module, c.name)
-            for c in classes
-            if c.category == category and not c.is_decorated and predicate(c)
+            (c.module, c.name) for c in classes if c.category == category and predicate(c)
```

Comment:
Not specifically related to this line, but I think this `add_silo_decorators` function should have a docstring explaining it's purpose.